client/my-sites/plans/features/google-analytics.js

```diff
@@ -27,10 +29,16 @@ export default function( context ) {
 	const freeLine = i18n.translate( 'Free for life' );
 	const freeButton = i18n.translate( 'Your Plan' );
 
+	const premiumTitle = i18n.translate( 'Premium' );
+	const premiumLine = i18n.translate( '$99 per year' );
+	const premiumButton = i18n.translate( 'Your Plan' );
```

Comment:
We should make the PlanCard component smart, but this fixes changes that are staged.